tests/CMakeLists.txt

```diff
@@ -511,19 +514,19 @@ add_test_case(test_uri_decode)
 
 generate_test_driver(${PROJECT_NAME}-tests)
 
-if (NOT MSVC AND NOT LEGACY_COMPILER_SUPPORT)
-    #we have some tests here that purposely overflow
+if(NOT MSVC AND NOT LEGACY_COMPILER_SUPPORT)
+    # we have some tests here that purposely overflow
```

Comment:
what plugin is enforcing all this whitespace consistency?

I like it

Reply:
I don't think I used anything special for cmake file format, it's probably just the default one with VScode. But, I do use [cbrevik.toggle-format-on-save](https://marketplace.visualstudio.com/items?itemName=cbrevik.toggle-format-on-save) to automatic format files.